src/cca.jl

```diff
@@ -336,3 +340,113 @@ function fit(::Type{CCA}, X::AbstractMatrix{T}, Y::AbstractMatrix{T};
 
     return M::CCA
 end
+
+abstract type MultivariateTest <: HypothesisTest end
```

Comment:
Perhaps it would be worthwhile for these types to be defined in HypothesisTests, then this package can simply provide methods for the constructors. Thoughts?

Reply:
I think there's some merit in this idea, but it also seems fine to have `MultivariateTest` live in `MultivariateStats`. @ararslan do you feel strongly enough about this that we should delay merging this?

Reply:
MultivariateTests was folded into HypothesisTests rather than MultivariateStats, so HypothesisTests already contains some multivariate tests that could make sense to be part of the same hierarchy, e.g. Hotelling's $T^2$ test. Would shuffling around the abstract type definitions after the fact be breaking?

Reply:
I think if this package re-exports them, it wouldn't be breaking. So should we merge this and decide later? That's work that could be done by anybody, we don't need to drag @kshedden through it.